server/src/test/java/org/apache/iotdb/db/engine/merge/MergeTest.java

```diff
@@ -145,11 +146,18 @@ void prepareFiles(int seqFileNum, int unseqFileNum)
 
   private void removeFiles() throws IOException {
     for (TsFileResource tsFileResource : seqResources) {
-      tsFileResource.remove();
-    }
+      try {
+          tsFileResource.remove();
+        }catch (NullPointerException e) {
+      }
```

Comment:
Why would an NPE rise? You should fix it instead of ignoring it.

Reply:
fixed~